alcs-frontend/src/app/shared/header/notifications/notifications.component.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../../styles/colors';
+
+.unread-notifications::before {
+  content: '';
+  position: absolute;
+  top: 13px;
+  background: $error-color;
+  width: 7px;
```

Comment:
I know one designer who will be furious to see 7px instead of 8px 😅

Reply:
The good news is I already got approval via screenshot haha